lib/couchbase_helper/documentgenerator.py

```diff
@@ -639,6 +642,27 @@ def read_from_dump(self):
             print ("Unable to find file lib/couchbase_helper/"
                        "geospatial/earthquakes.json, data not loaded!")
 
+    def custom_read_from_dump(self,filename="sample-data.json"):
+        count = 0
+        done = False
+        try:
+            with open("./lib/couchbase_helper/geospatial/"+filename) as f:
```

Comment:
make the directory path formation using "/"  platform independent. Use the more python recommend way of doing it.

Reply:
sample-data.json is used and if so, how ?

Reply:
`sample-data.json` is a default, in case a filename is not passed as part of the keyword args.